index.html

```diff
@@ -105,7 +105,7 @@
             ), and containerd via
             <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
             It's also used by
-            <a href="https://fly.io/">fly.io</a>
+            <a href="https://fly.io/">fly.io</a> as well as <a href="https://appfleet.com/">appfleet</a>
```

Comment:
I think the lines between "used by"/"integrated with" are quite blurry at this point. Initially, we tried to differentiate between open source integrations and other things, but that's just confusing now, I think.

So instead of
`
Firecracker is integrated with Kata Containers, Open Nebula, UniK, Weave FireKube (via Weave Ignite ), and containerd via firecracker-containerd. It's also used by fly.io as well as appfleet.
`

we could go with
`
Firecracker is used by/integrated with (in alpahbetical order): appfleet, containerd via firecracker-containerd, fly.io, Kata Containers, Open Nebula, UniK, and Weave FireKube (via Weave Ignite ).
`
What do folks think?

Reply:
I've just included these changes in https://github.com/firecracker-microvm/firecracker-microvm.github.io/pull/9 (also at the FAQ section)